wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -140,11 +145,17 @@ extension ZMConversation {
                     completion(.failure(ConversationFetchError.unknown))
                     return
                 }
-                let fetchResult = (conversationId, conversationName)
+
+                var passwordProtected: Bool?
+                if apiVersion.rawValue >= 4 {
```

Comment:
If `passwordProtected` will have a default value as false, do we really need to check the apiVersion?